### PR TITLE
initial crowdin config

### PR DESCRIPTION
**Pull request by maddhruv**
This is an initial crowdin config for the content structure you proposed in #61

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+preserve_hierarchy: true
+files:
+  # /content/{vodeVersion}/en-US/doc/{filesInSubFolders}
+  # markdown extensions - md, mkd, mkdn
+  - source: /content/v*/en-US/doc/**/.(m?d?|markdown)
```
Comment (zeke) [inline crowdin.yml]: `/content/v*/en-US/doc/**/*.md` should do the trick here. `md` appears to be the only extension used in the node docs.

**Comment by zeke** [comment]
@andrulko how does this look to you? When we set up the Electron project, we needed to first create this file in Crowdin's web interface. But maybe things have changed since then?

**Comment by maddhruv** [comment]
While creating a new project with crowdin, it syncs up things and the crowdin.yml file is the first thing it looks forward to

**Comment by maddhruv** [comment]
merging!
Hoping to work as fine :crossed_fingers:

**Pull request merged**